### PR TITLE
Update Free_for_Open_Source_Application_Security_Tools.md

### PR DESCRIPTION
**Pull request by weirdlantern**
Adding an opensource tool for github repository and contributor analysis

### CODE REVIEW

File: pages/Free_for_Open_Source_Application_Security_Tools.md [hunk 1]
```diff
@@ -348,6 +348,10 @@ Secrets detection is often confused with SAST because both scan through static s
...
 
+### Intel and Repository Analysis Tools
+
+- [Gitxray](https://github.com/kulkansecurity/gitxray/) - Gitxray (short for Git X-Ray) is an opensource tool designed for use on GitHub repositories. It leverages public GitHub REST APIs to gather information on Contributors and Repositories that would otherwise be very time-consuming to obtain manually. It can identify fake or shared contributor accounts, collect sensitive information in contributor profiles by looking in unconventional places (e.g. by parsing GPG key blobs), flag dangerous repository activity, and a lot more available in [its documentation](https://www.gitxray.com).
```
Comment (kingthorin) [inline pages/Free_for_Open_Source_Application_Security_Tools.md:353]: It's "short" as in dropped a space and a hyphen? That kinda seems like a difference without a distinction

Comment (weirdlantern) [reply pages/Free_for_Open_Source_Application_Security_Tools.md:353]: @kingthorin - We can get rid of that clarification if it does not add value.

Comment (kingthorin) [reply pages/Free_for_Open_Source_Application_Security_Tools.md:353]: If you really like it, it's fine. This is community content.
Just seemed kinda funny to me.

Comment (weirdlantern) [reply pages/Free_for_Open_Source_Application_Security_Tools.md:353]: Then let's leave it that way, easier, short for "Easier." :laughing:

**Pull request merged**